Remove sql in the footer

sujena2407 opened the pull request.

sujena2407 committed:
Remove sql

sujena2407 committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

sujena2407 committed:
Icon added instagram

Abi1327 committed:
Update styles.css

Changed the clr

Ramesh623 committed:
Added my profile to the developer section

Ramesh623 committed:
Updated the timeline

The Blog Website part removed from the timeline and replace that space using Technical Training content

neranjanpriyashantha committed:
Added ML in python and DA in python

Added "Python for Machine Learning" to python blog
Added "Data Analytics Using Python" to python blog
Added data-analytics.png to img folder
Added class "text-align-left" to style.css

pankajan05 committed:
Merge branch 'Master' into Dev

pankajan05 commented on css/styles.css:
```diff
@@ -11408,7 +11408,7 @@ header.masthead {
   text-align: center;
   color: #0d3b25d4;
-  background-color: #0a7512d4;
+  background-image:linear-gradient(to right, #0a7512d4,#96CF24);
```
Good!

pankajan05 commented on index.html:
```diff
@@ -146,8 +146,8 @@ <h4 class="my-3">Data Analytics </h4>
         <section class="page-section bg-light" id="portfolio">
             <div class="container">
                 <div class="text-center">
-                  <h2 class="section-heading text-uppercase">Technologies Used</h2>
```
It needs to be Technologies used not a portfolio

pankajan05 replied:
@Abivarna Please change it to Technologies.

pankajan05 reviewed (commented):
please do the mentioned change to merge this pull request.

neranjanpriyashantha committed:
Added css animation to banner

Added header-bubble-style.css to css folder
Added bubble div 1-12 to index.html

neranjanpriyashantha committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

thenukagunawardana committed:
Added MERN blog and Changed the Order it appears on the Portfolio

thenukagunawardana committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

thenukagunawardana committed:
Elzian Agro Logo Added

Abi1327 committed:
Update index.html

added the icons

neranjanpriyashantha committed:
Added animation to banner

neranjanpriyashantha committed:
Added emailjs  cdn again

neranjanpriyashantha committed:
Changed title as h2 in python blog

pankajan05 commented on index.html:
```diff
@@ -5,35 +5,24 @@
         <meta name="description" content="" />
         <meta name="author" content="" />
-        <title>Elzian Agro - Developers Blog</title>
+        <title>Agency - Start Bootstrap Theme</title>
```
The Title should be Elzian Agro -Developer Blog. @Abivarna Please change this back.

pankajan05 commented on index.html:
```diff
@@ -5,35 +5,24 @@
         <!-- Core theme CSS (includes Bootstrap)-->
         <link href="css/styles.css" rel="stylesheet" />
-        <link rel="stylesheet" href="./css/header-bubble-style.css">
-        <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@emailjs/browser@3/dist/email.min.js"></script>
```
@Abivarna This is don't remove.

Abi1327 committed:
Title Changed

Abi1327 committed:
Changed the heading portfolio as technologies used

Abi1327 committed:
Changed Elzian Agro logo

Abi1327 committed:
Changed the subheading of Technology used

thenukagunawardana committed:
Updated Elzian Logo and Navbar

Changed the  navbar "Portfolio" to "Technologies"

pankajan05 reviewed (approved):
checked
